.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           - distro: centos-8
             env: py36
           - distro: centos-8
-            env: py38
+            env: py39
```

Comment:
As we are removing Python 3.8 from the CIs (and technically dropping support), we could remove `*-py38` from the available `tox` environments. Let's keep it for now and decide if we want to remove it in a follow-up patch.